tests/test_input.py

```diff
@@ -0,0 +1,28 @@
+"""
+Test cases for the input module.
+"""
+
+import unittest
+import networkx as nx
+from probinet.input.loader import build_adjacency_from_networkx
+from probinet.models.classes import GraphData
+from .fixtures import BaseTest
```

Comment:
Imports are not sorted properly.

Reply:
I will get this once it's merged, as we discussed.